.github/workflows/apidoc.yml

```diff
@@ -17,8 +17,12 @@ defaults:
     # to improve debugging some of the longer scripts.
     shell: /bin/bash -o errexit -o pipefail -o xtrace {0}
 
+permissions: {}
 jobs:
   main:
+    permissions:
+      contents: write # for git push
```

Comment:
As far as I'm aware, `GITHUB_TOKEN` is scoped to the repository that the workflow runs in. And this workflow never pushes there.
It pushes to https://github.com/awesomeWM/apidoc/.

Reply:
I was confused by `secrets.AWESOME_ROBOT_TOKEN || github.token`. My mistake.